README.md

```diff
@@ -4,10 +4,7 @@
 
```

Comment:
- [ ] In your README, kindly change the title from the generic **`Project Name`** to something more descriptive of the project you just built.

  ![karam-proj](https://user-images.githubusercontent.com/36542986/160041628-28f190af-d82e-4b20-b17e-e913669732da.jpg)

- [ ] In your README, kindly add detailed instructions on how users can set up and run the app locally i.e. from cloning the repo to running `npm install` (for installing packages) to running the `npm start` (to start the app)